.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+indent_style = tab
```

Comment:
We want spaces. ;) Please also check all files if they use spaces for indentation.